extra-devel/bcc/autobuild/build

```diff
@@ -0,0 +1,14 @@
+mkdir build
```

Comment:
So why not change `SUBDIR` in spec? Also, use absolute paths here.

Reply:
Note that this file exists solely to workaround the issue where PKGVER isn't defined in defines. Will use absolute paths though.